src/services/textChanges.ts

```diff
@@ -178,11 +194,12 @@ namespace ts.textChanges {
 
     function getAdjustedEndPosition(sourceFile: SourceFile, node: Node, options: ConfigurableEnd) {
         const { end } = node;
-        if (options.useNonAdjustedEndPosition || isExpression(node)) {
+        const { endPosition } = options;
+        if (endPosition === TrailingTriviaOption.Exclude || (isExpression(node) && endPosition !== TrailingTriviaOption.Include)) {

```

Comment:
This is the only place I can see `TrailingTriviaOption.IncludeIfLineBreak` being consumed and I don't understand how it relates to expressions.

Reply:
I don't know why it is necessary to check if `node` is an expression, but for the behavior I needed for this function (when the `TrailingTriviaOption.Include` is used), it should not matter if `node` is an expression. So I just tried not to break existing code. The same goes for the `isLineBreak` check below, I don't know what is the reasoning behind it.

Reply:
I think you got rid of `IncludeIfLineBreak`.